src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -43,12 +43,11 @@
 using namespace InferenceEngine;
 using namespace std;
 using namespace ov::intel_gna;
```

Comment:
According to gna_graph_compiler.hpp the content of this file should be contained in ov::intel_gna namespace, not using it.

Reply:
Will be fixed in next PR.